src/components/10-atoms/button-link/index.js

```diff
@@ -75,11 +74,12 @@ class AXAButtonLink extends LitElement {
         @click="${this.onClick}"
       >
         <div class="a-button-link__flex-wrapper">
-          <slot></slot>
-          ${icon &&
-            html`
-              <axa-icon class="a-button-link__icon" icon="${icon}"></axa-icon>
-            `}
+          <slot></slot> ${
+            icon &&
+              html`
+                <axa-icon class="a-button-link__icon" icon="${icon}"></axa-icon>
```

Comment:
Attention! This writes "false" to the DOM if icon is falsy.

Reply:
This is not my code, prettier formated it like that so i dont know the logic